platforms/jvm/plugins-java/build.gradle.kts

```diff
@@ -34,13 +34,23 @@ dependencies {
     implementation(libs.guava)
     implementation(libs.inject)
 
+    testFixturesImplementation(project(":internal-integ-testing"))
+    testFixturesImplementation(project(":logging"))
+
     testImplementation(testFixtures(project(":core")))
 
+    integTestImplementation(testFixtures(project(":enterprise-operations")))
+    integTestImplementation(testFixtures(project(":language-java")))
+    integTestImplementation(testFixtures(project(":language-jvm")))
+    integTestImplementation(testFixtures(project(":plugins-java-base")))
+    integTestImplementation(testFixtures(project(":workers")))
+
+    testRuntimeOnly(project(":distributions-jvm")) {
```

Comment:
Were we able to move the project builder tests to integ tests? 
We should avoid this distribution dependency for unit tests